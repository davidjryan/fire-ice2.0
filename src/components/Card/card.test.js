import { shallow } from 'enzyme';
import React from 'react';
import { Card } from './MovieCard';

describe('tests for Card component', () => {
  describe('component tests', () => {
    let cardWrapper;
    let mockProps;

    beforeEach(() => {
      mockProps = {
        
      };
      cardWrapper = shallow(<Card {...mockProps} />);

    });

    it('should match the snapshot', () => {
      expect(cardWrapper).toMatchSnapshot();
    });
  });
});