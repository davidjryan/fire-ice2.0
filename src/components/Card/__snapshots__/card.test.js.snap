// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Card component component tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    cardData={
        Object {
            "houseData": Object {
              "ancestralWeapons": Array [
                "Lady Forlorn",
              ],
              "cadetBranches": Array [],
              "coatOfArms": "Three black ravens in flight, holding three red hearts, on a white field(Argent, three ravens volant sable, each clutching in their claws a heart gules)",
              "currentLord": "https://www.anapioficeandfire.com/api/characters/681",
              "diedOut": "",
              "founded": "",
              "founder": "https://www.anapioficeandfire.com/api/characters/256",
              "heir": "https://www.anapioficeandfire.com/api/characters/677",
              "name": "House Corbray of Heart's Home",
              "overlord": "https://www.anapioficeandfire.com/api/houses/7",
              "region": "",
              "seats": Array [
                "Heart's Home",
              ],
              "swornMembers": Array [
                "https://www.anapioficeandfire.com/api/characters/255",
                "https://www.anapioficeandfire.com/api/characters/256",
                "https://www.anapioficeandfire.com/api/characters/447",
                "https://www.anapioficeandfire.com/api/characters/654",
                "https://www.anapioficeandfire.com/api/characters/677",
                "https://www.anapioficeandfire.com/api/characters/681",
                "https://www.anapioficeandfire.com/api/characters/871",
                "https://www.anapioficeandfire.com/api/characters/1182",
              ],
              "titles": Array [
                "Lord of Heart's Home",
                "King of the Fingers (historical)",
              ],
              "url": "https://www.anapioficeandfire.com/api/houses/81",
              "words": "",
            },
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="title"
/>,
        <p
          className="title"
/>,
        <p
          className="title"
/>,
        <p
          className="title"
/>,
        <p
          className="title"
/>,
        <p
          className="title"
/>,
        <p
          className="title"
/>,
      ],
      "className": "Card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "title",
        },
        "ref": null,
        "rendered": null,
        "type": "p",
      },
    ],
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="title"
/>,
          <p
            className="title"
/>,
          <p
            className="title"
/>,
          <p
            className="title"
/>,
          <p
            className="title"
/>,
          <p
            className="title"
/>,
          <p
            className="title"
/>,
        ],
        "className": "Card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "title",
          },
          "ref": null,
          "rendered": null,
          "type": "p",
        },
      ],
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
